src/App/Osu/OsuBackgroundStarCacheLoader.cpp

```diff
@@ -67,10 +67,19 @@ void OsuBackgroundStarCacheLoader::initAsync()
 		double speedDifficultStrains = 0.0;
 
 		// new fast method  (build full cached DiffObjects once) (1/2)
+		double cacheObjectsStart = engine->getTimeReal();
```

Comment:
Generally engine calls are not allowed async (not threadsafe), this can collide with a parallel call on the main thread. Just allocate a Timer temporarily on the stack for that scope.